perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/MailManagerImpl.java

```diff
@@ -120,7 +123,7 @@ protected void initialize() throws PerunException {
 
 	@Override
 	@Transactional(rollbackFor = Exception.class)
-	public Integer addMail(PerunSession sess, ApplicationForm form, ApplicationMail mail) throws PerunException, DuplicateKeyException {
+	public Integer addMail(PerunSession sess, ApplicationForm form, ApplicationMail mail) throws PerunException {
```

Comment:
Can you please convert this to throw specific exceptions instead of PerunException ? Thank you.